dist/images/ovndb-vip.sh

```diff
@@ -65,12 +64,12 @@ check_ovn_daemonset_version () {
 # create the ovnkube_db endpoint for other pods to query the OVN DB IP
 create_ovnkube_db_ep () {
   # delete any endpoint by name ovnkube-db
-  kubectl --server=${K8S_APISERVER} --token=${k8s_token} --certificate-authority=${K8S_CACERT} \
+  kubectl --token=${k8s_token} --certificate-authority=${K8S_CACERT} \
```

Comment:
You shouldn't need to pass `--token` or `--certificate-authority` either. `kubectl` ought to be able to read them from the mounted secret, just like the script does.